definitions/features/upstream_repositories.rb

```diff
@@ -0,0 +1,62 @@
+class Features::UpstreamRepositories < ForemanMaintain::Feature
+  metadata do
+    label :upstream_repositories
+
+    confine do
+      !feature(:instance).downstream?
+    end
+  end
+  attr_accessor :version
```

Comment:
Repositories setup is based on version value. Here - https://github.com/theforeman/foreman_maintain/pull/356/files#diff-a2969a5900d684668da14e17f0a8d7fcd9531ac7a3bab6ee8e2a23ded4ca08b1R17
it is expected to pass an argument i.e `version`. 
Wherever needed you already passed `version` as method argument. It is better not to declare `attr_accessor :version` and pass `version` value to nested methods.